### ISSUE TITLE
Page crashes on mobile (android, iOS) when loading long audio file

### ISSUE THREAD

**Issue by estafaa** [processing/p5.js#4209]
#### Nature of issue?

- [x] Found a bug

#### Most appropriate sub-area of p5.js?

- [x] Sound

#### Which platform were you using when you encountered this?

- [x] Mobile/Tablet (touch devices)

#### Details about the bug: 

- p5.js version: 0.10.2 
- Web browser and version: chrome mobile 79.0.3945.93 (official built) (32-bit), 
- Operating System: Android 10
- Steps to reproduce this:

```
var song;
var songUrl = "https://aquapsy.s3.ca-central-1.amazonaws.com/store/c12f652ea5855acf0c1f5d9302f17f26.mp3"; 
 // it is about 37 minutes long song

function preload(){
    song = loadSound(songUrl, playsound,'', whileLoading);
    };

function playsound(e){
};

function whileLoading(e){
};

someButton.addEventListener("click", function(){
  song.play();
});
```

Everything works fine on a laptop on different browsers (songs can be long). 
And everything works well on laptop and mobile/tablet if a song is **short** (I noticed that important not the size of the file, but duration). 10 minutes song is okay, but that 37 minutes song on mobile/tablet after the sound is loaded the page crashes and reloads. On android, it says: "Aw, Snap! Something went wrong while displaying this webpage.." 
Seems after loadSound() is finished, it crashes, I do not start play or anything like that.

What is the problem?

Thank you very much!

**Comment by isaac-computer** [comment]
@estafaa Yes! I have the same problem but haven't seen anyone else mentioning it. I don't know of a solution yet. Did you ever figure anything out?
